### ISSUE TITLE
Looks great but not docs

### ISSUE THREAD

**Issue by muka** [etorras/PSPointsBundle#10]
Hello, I'm going to try the bundle and hopefully contribute on docs

What about creating a basic documentation to setup and use?

Thank you

**Comment by etorras** [comment]
You're right @muka! I never finished this bundle. If you like to contribute, you're welcome!
